SourceCode/support/convert_masterfiles_to_csv.py

```diff
@@ -115,24 +172,30 @@ def get_remaining_variables(vars_to_convert, out_dir, model, \
                     gamma_input_on_overwrite(out_dir, var, gamma_options)
         
         return vars_to_convert, gamma_options
+    
     vars_to_convert_remaining = []
-    for var in vars_to_convert:
-        
+    for var in vars_to_convert:     
         out_fn = os.path.join(out_dir, f"{var}.csv")
         out_fn_reg = os.path.join(out_dir, f"{var}_BE.csv")
+        
+        # Condition 1
         exists = (csv_exists(out_fn) or csv_exists(out_fn_reg))
                 
         if not exists:
             vars_to_convert_remaining.append(var)
+            
+    
     return vars_to_convert_remaining, gamma_options
 
 
 def gamma_input_on_overwrite(out_dir, var, gamma_options):
-    """ When the script is run as a script, and
+    """
+    When the script is run as a script, and
     the gamma csv files already exist, confirm with
     the user if you want to overwrite, given that
     the user may not want to lose their calibrated gamma
-    values """
+    values 
+    """
     
     costvar_to_gam_dict = {"BTTC": "TGAM", "BHTC": "HGAM", "ZCET": "ZGAM"}
```

Comment:
I thought a simple fix to add functionality for the Power module would be to update this list and the corresponding "GAMMA" cell in FTT_variables.csv. However, this has not worked and produces an error

Reply:
Furthermore, the gamma csvs are still being overwritten for the non-power modules, even after inputting "n" to avoid overwrite